plugins/storage/pyparts/ceph_daemon_checks.py

```diff
@@ -160,6 +160,10 @@ def get_ceph_pg_imbalance(self):
             issue = issue_types.CephCrushWarning(msg)
             issue_utils.add_issue(issue)
 
+    @staticmethod
+    def version_compare(ver):
```

Comment:
Can you name this in a way that better explains what it is to be used for and also a docstring and associated unit test. If you think this is a good way to compare ceph daemon version it might be better to put it somewhere like https://github.com/canonical/hotsos/blob/6a0d650a8d76b5a5f85f4ddc8c0a9f8939e1de7a/core/plugins/storage/ceph.py#L186

Reply:
this method returns a tuple of ints of all the numbers in a version , so we can compare using < or > . version_compare could probably be called "return_version_as_tuple" perhaps. I could put it in the CephDaemonBase if that's a better place for it, I will discuss further and then make the changes as required.